lib/src/pdf_page_proxy.dart

```diff
@@ -18,33 +18,48 @@ class PDFPageProxy {
   JsObject _jsInternal;
 
   PDFPageProxy() {
-    _jsInternal = JsObject(context['pdfjsLib']['PDFPageProxy']);
+    checkNotNull(context, message: 'context was null in PDFPageProxy');
```

Comment:
#nit Use `ArgumentError.checkNotNull` instead of bringing in a dependency just for null checks